wcfsetup/install/files/lib/system/event/listener/AbstractDynamicDatabaseObjectEventListener.class.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace wcf\system\event\listener;
+use wcf\data\AbstractDatabaseObjectAction;
+
+/**
+ * Special event listener for `\wcf\data\AbstractDatabaseObjectAction` events.
+ * This appends behind the called function name the action of the database object
```

Comment:
```suggestion
 * When executed for the validateAction or finalizeAction event it will dispatch to a function specific
 * to the current action value of the AbstractDatabaseObjectAction.
```